server.py

```diff
@@ -0,0 +1,15 @@
+from flask import Flask, render_template
+
+app = Flask(__name__)
+
+
+@app.route("/")
+def desk():
+    return render_template("index.html")
+
+
+# def ask(word):#
+
+
+if __name__ == "__main__":
+    app.run(host='0.0.0.0', port=5000)

```

Comment:
```suggestion
    app.run()
```
I think you don't need to set that.